src/devices/CharacterLcd/CharacterLcd.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
please remove sln file

Reply:
I actually left it in due to #784, but I guess the decission on where to proceed there is still open.

Reply:
ok, fine to leave since we have a tracking issue to clarify this